soroban-env-guest/src/guest.rs

```diff
@@ -161,6 +165,11 @@ impl EnvBase for Guest {
         // Intentionally a no-op in this cfg. See above.
         Ok(())
     }
+
+    fn prng_fill_slice(&self, mem: &mut [u8], finalize: bool) -> Result<(), Status> {
+        // Intentionally a no-op in this cfg. See above.
+        Ok(())
```

Comment:
❓ Why is this a no-op in the guest? We'd want this to call through to the `prng_fill_linear_memory` function wouldn't we, like how the bytes functions do above.

Reply:
Oops, correct. Will fix.